assets/js/translator.js

```diff
@@ -18,6 +18,31 @@ if(modeEnabled('translation')) {
     $(document).ready(function () {
         synchronizeTextareaHeights();
 
+        if(config.TRANSLATION_CHAINING) {
+            $('.chainedTranslation').show()
+            function getTargets (src) {
```

Comment:
Let's go with `getDstLanguages(src)`, i.e. use destination/dst instead of target since that's what the rest of the code uses.